src/sentry/auth/helper.py

```diff
@@ -450,6 +450,7 @@ def handle_unknown_identity(request, organization, auth_provider, provider, stat
                 request,
                 {
                     "identity": identity,
+                    "provider": auth_provider.provider.capitalize() if auth_provider else " ",
```

Comment:
damn we really dont have a name attribute/helper? this will be "Github" vs "GitHub"

Reply:
good point. updated with the appropriate name attribute.